nifi/Dockerfile

```diff
@@ -89,5 +66,55 @@ COPY shared/log4shell_scanner /bin/log4shell_scanner
 RUN /bin/log4shell_scanner s /stackable/nifi-${PRODUCT}
 # ===
 
+# Final image
+FROM stackable/image/java-base
+
+ARG PRODUCT
+ARG RELEASE
+
+LABEL name="Apache NiFi" \
+      maintainer="info@stackable.tech" \
+      vendor="Stackable GmbH" \
+      version="${PRODUCT}" \
+      release="${RELEASE}" \
+      summary="The Stackable image for Apache NiFi." \
+      description="This image is deployed by the Stackable Operator for Apache NiFi."
+
+RUN microdnf update && \
+    microdnf install \
+    python-devel \
+    python-pip \
+    python-setuptools && \
+    microdnf clean all \
+    && rm -rf /var/cache/yum
+
+RUN pip install --no-cache-dir nipyapi==0.19.1
+
+# For backwards compatibility we create a softlink in /bin where the jar used to be as long as we are root
+# This can be removed once older versions / operators using this are no longer supported
+RUN ln -s /stackable/stackable-bcrypt.jar /bin/stackable-bcrypt.jar
```

Comment:
just for correctness I would put this after line 100

Reply:
But that hadolint warning is valid. This being the final image it probably makes sense to put the RUN things together?

Reply:
tried to use less run statements https://github.com/stackabletech/docker-images/pull/678/commits/8c2e491933bebaa6f9e0f5c5f79a0f7003e41450